### DISCUSSION TITLE
ideas/suggestions

### DISCUSSION THREAD

**Discussion by whatotter** [whatotter/pwnhyve#3] [Ideas]
ideas or suggestions to add to pwnhyve; can be from p4wnp1 or from pwnagotchi, or anything really

**Comment by ghost** [comment]
please make a video about it i wanna know wat is that

**Comment by NoobReider** [comment]
Can i use waveshare 1,44" instead of 1,3"

**Comment by shellnowasklater** [comment]
I own a zero w that I've been throwing p4 aloa on as well as pwnagotchi I ran into use cases with the pwangochi which was nice but it seemed to have limitations I then added my own shoddy python script to p4wnp1 aloa to kind of be a multi tool however I ran into issues with the spec witlessadapter usng monitor mode. How would I go about adding this project and ALOA together since I have a usb on it it's nice to have as both use cases.

**Comment by raptor0999** [comment]
Hi there! I just discovered this pwnhyve and have been checking it out and wanting to build one especially since I already have quite a few pi zeros sitting around!

I was wondering would it be easy enough to add on NFC as a plugin? I am a developer so don't mind writing it myself if I can, just wanted to know if I would be able to essentially add a NFC hat or otherwise and use a plugin to be able to read NFC and RFID tags with it

Thanks for the work on pwnhyve already!

**Reply by whatotter** [reply]
you totally could add NFC/RFID, the biggest issue is finding where to put the hardware, since pwnhyve is so space-constrained

i recommend designing something around pwnhyve's PCB shim, since it gives you outputs to add modules exactly like the flipper zero, which you can then develop an NFC module as an add-on board. going this route, you do have to design a PCB though, which i'm not sure if your comfortable with doing

i would be completely down to help with this too :) messing with NFC is fun
